lib/ambidextrous.rb

```diff
@@ -1,20 +1,51 @@
 module Ambidextrous
-  IPHONE_APP_USER_AGENT_PATTERN_1 = /Titanium/i
   IPHONE_APP_USER_AGENT_PATTERN_2 = /^iNaturalist\/\d+.+iOS/i
-  IPHONE_APP_USER_AGENT_PATTERN = /#{IPHONE_APP_USER_AGENT_PATTERN_1}|#{IPHONE_APP_USER_AGENT_PATTERN_2}/
-  IPHONE_APP_USER_AGENT_PATTERNS = [IPHONE_APP_USER_AGENT_PATTERN, IPHONE_APP_USER_AGENT_PATTERN_2]
-  ANDROID_APP_USER_AGENT_PATTERN = /^iNaturalist\/\d+.+Android/i
-  INATRN_APP_USER_AGENT_PATTERN = /^iNaturalistRN/
-  MOBILE_APP_USER_AGENT_PATTERNS = [IPHONE_APP_USER_AGENT_PATTERNS, ANDROID_APP_USER_AGENT_PATTERN].flatten
-
-  FISHTAGGER_APP_USER_AGENT_PATTERN = /fishtagger/i
-  
+
```

Comment:
rubocop is suggesting these constants should be frozen (adding `.freeze`) which seems like a worthwhile addition